api/v0/discovery/discovery.proto

```diff
@@ -1,5 +1,5 @@
 /* -------------------------------------------------------------------------- *\
- *        Apache 2.0 License Copyright © 2022-2023 The Aurae Authors          *
+ *             Apache 2.0 License Copyright © 2022 The Aurae Authors          *
```

Comment:
I know we have the upcoming PR to remove dates, but do we want to leave it with the range for now.

Reply:
this was a mistake. reverted.